engines/gob/detection/tables.h

```diff
@@ -49,6 +49,7 @@ static const PlainGameDescriptor gobGames[] = {
 	{"lit", "Lost in Time"},
 	{"lit1", "Lost in Time Part 1"},
 	{"lit2", "Lost in Time Part 2"},
+	{"nathanvacances", "Nathan Vacances"},
```

Comment:
Since you've added a new game target here, it should be added in the other relevant engine places as well (add a game ID, add a mapping in `gameidtotype.h`) etc

Reply:
so adding also kGameTypeNathanVacances, if so i would do it also for Adibou 3 and Adi 5 and all the other DEV7 games